main/src/main/scala/sbt/internal/server/NetworkChannel.scala

```diff
@@ -613,7 +613,7 @@ final class NetworkChannel(
   def logMessage(level: String, message: String): Unit = {
     import sbt.internal.langserver.codec.JsonProtocol._
     jsonRpcNotify(
-      "window/logMessage",
+      "build/logMessage",
```

Comment:
Could you also update the corresponding 
network client handling? - https://github.com/sbt/sbt/blob/1f3ac6c8cd6fc38f8a6ad85ddca53ade346a7d34/main-command/src/main/scala/sbt/internal/client/NetworkClient.scala#L136